File: app/src/test/java/com/hana053/micropost/ui/pages/top/TopActivityTest.java
package com.hana053.micropost.ui.pages.top;

import com.hana053.micropost.services.AuthTokenService;
import com.hana053.micropost.testing.RobolectricBaseTest;
import com.hana053.micropost.testing.RobolectricDaggerMockRule;

import org.junit.Before;
import org.junit.Rule;
import org.junit.Test;
import org.mockito.Mock;
import org.robolectric.Robolectric;
import org.robolectric.util.ActivityController;

import static org.hamcrest.Matchers.instanceOf;
import static org.junit.Assert.assertThat;

public class TopActivityTest extends RobolectricBaseTest {

    @Rule
    public final RobolectricDaggerMockRule rule = new RobolectricDaggerMockRule();

    @SuppressWarnings("FieldCanBeLocal")
    private ActivityController<TopActivity> activityController;
    private TopActivity activity;

    @Mock
    AuthTokenService authTokenService;

    @Before
    public void setup() {
        activityController = Robolectric.buildActivity(TopActivity.class);
        activity = activityController.setup().get();
    }

    @Test
    public void shouldBeCreated() {
        assertThat(activity, instanceOf(TopActivity.class));
    }

}

File: app/src/test/java/com/hana053/micropost/activity/pages/top/TopActivityTest.kt
package com.hana053.micropost.activity.pages.top

import com.hana053.micropost.pages.top.TopActivity
import com.hana053.micropost.testing.RobolectricBaseTest
import org.hamcrest.Matchers.instanceOf
import org.junit.Assert.assertThat
import org.junit.Before
import org.junit.Test
import org.robolectric.Robolectric
import org.robolectric.util.ActivityController

class TopActivityTest : RobolectricBaseTest() {

    lateinit var activityController: ActivityController<TopActivity>
    lateinit var activity: TopActivity

    @Before
    fun setup() {
        activityController = Robolectric.buildActivity(TopActivity::class.java)
        activity = activityController.setup().get()
    }

    @Test
    fun shouldBeCreated() {
        assertThat(activity, instanceOf(TopActivity::class.java))
    }

}